universum/lib/utils.py

```diff
@@ -93,21 +93,21 @@ def check_required_option(settings, setting_name, error_message):
 def read_and_check_multiline_option(settings, setting_name, error_message):
     try:
         value = getattr(settings, setting_name)
+        if value.startswith('@'):
+            try:
+                with open(value.lstrip('@')) as file_name:
+                    result = file_name.read()
+            except FileNotFoundError as e:
+                raise IncorrectParameterError(f"Error reading argument {setting_name} from file {e.filename}: no such file")
+        elif value == '-':
+            result = sys.stdin.read()
+        else:
+            result = value
+        if not result:
+            raise AttributeError('empty value')
```

Comment:
I think raising IncorrectParameterError has the benefit of keeping original stack trace and is cleaner.